ygot/render.go

```diff
@@ -283,6 +455,40 @@ func leavesToNotifications(leaves map[*path]interface{}, ts int64, prefix []inte
 	return []*gnmipb.Notification{n}, nil
 }
 
+// stripPrefix removes the prefix pfx from the supplied path, and returns the more
+// specific path elements of the path. If the prefix is invalid for the path an
+// error is return.
+func stripPrefix(path *gnmiPath, pfx *gnmiPath) (*gnmiPath, error) {
+	if path.isStringSlicePath() != pfx.isStringSlicePath() {
```

Comment:
Optional: !path.isSameType(pfx)
It may seem like overkill, but the casual reader does not know that the line above is equivalent for gnmiPath type without digging.